CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# 92.2.0
+
+This minor hotfix release adds new features to MDCAlertController and MDCTabBarView.
+
+## New features
+
+MDCAlertController has a new `accessoryView` property. This property can be used to add custom views
+to the alert controller. Example usage:
+
+```swift
+let alert = MDCAlertController(title: "This is a title", message: "This is a message")
+let textField = UITextField()
+textField.placeholder = "This is a text field"
+alert.accessoryView = textField
+``` 
+
+TabBarView (a Beta component) has a new `setContentPadding:forLayoutStyle:` API. Example usage:
+
+```objc
+tabBarView.preferredLayoutStyle = MDCTabBarViewLayoutStyleScrollable;
+[tabBarView setContentPadding:UIEdgeInsetsMake(0, 8, 0, 8)
+               forLayoutStyle:MDCTabBarViewLayoutStyleScrollable];
+``` 
+
+## Changes
+
+### Dialogs
+
+* [Add accessoryView to MDCAlertController (#8538)](https://github.com/material-components/material-components-ios/commit/c84e1ab11535a2dd846e0500374883bcd9044edc) (Jacob Lukas‎)
+
+### Tabs
```

Comment:
Tip: Please revert this from the hotfix - it isn't a pressing issue and I'd rather not have it risk getting rolled-back because it's in with a hotfix.

Reply:
Done.